pulp-api/signing.py

```diff
@@ -0,0 +1,20 @@
+from pulpcore.app.models.content import AsciiArmoredDetachedSigningService
+
+NAME="sign-metadata-v1"
+
+# read an already exported public key
+with open("/tmp/public.key") as key:
+    with open("/tmp/public.fpr") as fpr:
+        fingerprint=fpr.read().rstrip('\n')
+        service_name = f"{NAME}-{fingerprint}"
+        try:
+          AsciiArmoredDetachedSigningService.objects.get(name=service_name)
+          print(f"Signing service {service_name} already present")
```

Comment:
The `printf` should be in an `else:` block (try/except/else/finally) since it's expected to only run if no exception.